tests/sources/xml/test_datacite.py

```diff
@@ -359,6 +380,136 @@ def test_datacite_with_attribute_and_subfield_variations_transforms_correctly():
     )
 
 
+def test_get_alternate_titles_success():
+    source_record = create_datacite_source_record_stub(
+        """
+        <titles>
+         <title>The Impact of Maternal Literacy and Participation Programs</title>
+         <title titleType="AlternativeTitle">An Alternative Title</title>
+         <title titleType="Subtitle">Baseline Data</title>
+        </titles>
+        """
+    )
+    assert Datacite.get_alternate_titles(source_record) == [
+        AlternateTitle(value="An Alternative Title", kind="AlternativeTitle"),
+        AlternateTitle(value="Baseline Data", kind="Subtitle"),
+    ]
+
+
+def test_get_alternate_titles_transforms_correctly_if_fields_blank(
+    datacite_record_optional_fields_blank,
+):
+    assert Datacite.get_alternate_titles(datacite_record_optional_fields_blank) == []
+
+
+def test_get_alternate_titles_transforms_correctly_if_fields_missing(
+    datacite_record_optional_fields_missing,
+):
+    assert Datacite.get_alternate_titles(datacite_record_optional_fields_missing) == []
+
+
+def test_get_content_type_success():
+    source_record = create_datacite_source_record_stub(
+        """
+        <resourceType resourceTypeGeneral="Dataset">Survey Data</resourceType>
+        """
+    )
+    assert Datacite.get_content_type(source_record, "abc123") == ["Dataset"]
+
+
+def test_get_content_type_transforms_correctly_if_fields_blank(
+    datacite_record_optional_fields_blank,
+):
+    assert (
+        Datacite.get_content_type(datacite_record_optional_fields_blank, "abc123") == []
+    )
```

Comment:
Curious what would happen if we still used the stub, but passed it nothing?  Would the effect be the same?  Again, not a request, but per the PR notes, just some high level thinking and questions in this early PR.

Reply:
A good suggestion, `blank` would need something passed but `missing` wouldn't, that would save us some fixtures in `conftest.py`. If @jonavellecuerdo agrees with that approach, I think I'll make the change

Reply:
This approach sounds good to me! I used this [method in updates to `SpringshareOaiDc` unit tests](https://github.com/MITLibraries/transmogrifier/pull/177/files#diff-9b1d193a2a05c09280b7b741221b67f2431bc44c9b4c69c3f79f09357314e8a3R131-R138) as well!